doc/classes/Array.xml

```diff
@@ -227,16 +242,35 @@
 			<description>
 				Returns [code]true[/code] if the array contains the given value.
 				[codeblock]
+				[gdscript]
 				print(["inside", 7].has("inside")) # True
 				print(["inside", 7].has("outside")) # False
 				print(["inside", 7].has(7)) # True
 				print(["inside", 7].has("7")) # False
+				[/gdscript]
+				[csharp]
+				var arr = new Godot.Collections.Array{"inside", 7};
+				// has is renamed to Contains
+				GD.Print(arr.Contains("inside")); // True
+				GD.Print(arr.Contains("outside")); // False
+				GD.Print(arr.Contains(7)); // True
+				GD.Print(arr.Contains("7")); // False
+				[/csharp]
 				[/codeblock]
 				[b]Note:[/b] This is equivalent to using the [code]in[/code] operator as follows:
 				[codeblock]
+				[gdscript]
 				# Will evaluate to `true`.
 				if 2 in [2, 4, 6, 8]:
 				    pass
+				[/gdscript]
+				[csharp]
+				// As there is no "in" keyword in C#, you have to use Contains
+				var array = new Godot.Collections.Array{2, 4, 6, 8};
+				if array.Contains(2) {
+				    // is true
```

Comment:
Several problems. One, `if` statements always need `()`. Two, the brace style is wrong.

```suggestion
				if (array.Contains(2))
				{
				    // is true
```

Reply:
fixed